components/commons/overlay-coretokens.css

```
@@ -11,9 +11,9 @@ governing permissions and limitations under the License.
 */
 
 %spectrum-overlay {
-  /* TODO: replace with core tokens */
-  --spectrum-overlay-animation-distance: 6px;
+  --spectrum-overlay-animation-distance: var(--spectrum-animation-distance);
```

```suggestion
  --spectrum-overlay-animation-distance: 6px;
```
Isn't `--spectrum-animation-distance` only defined in the Overlay index.css? I don't see it as a global token. I see other components like Tooltip only importing the _commons_ file.

---

Won't `--spectrum-animation-distance` still be undefined here?